js/models/message_handler.js

```diff
@@ -14,6 +14,13 @@ export default class MessageHandler {
     this.aladin.gotoRaDec(msg["ra"], msg["dec"]);
   }
 
+  handleAddHips(msg) {
+    const options = convertOptionNamesToCamelCase(msg["options"] || {});
+    const hips = A.imageHiPS(msg["src"], options);
+    if (msg["src"].includes("http")) hips.url = msg["src"];
```

Comment:
Why do you need to test if it is an url ? The first parameter of A.imageHiPS accepts either an ID or an url so it should work.

Reply:
When giving an url as the first argument of A.imageHiPS, the ImageHiPS object contain undefined for the url field (so aladin.addNewImageLayer don't find the survey)

Reply:
In this example it is working: https://jsbin.com/kufenesati/edit?html,output
Are you using the beta (3.4.1-beta) ?

Reply:
Indeed, AL 3.4.1-beta fixes this bug !